presto-product-tests/bin/run_on_docker.sh

```diff
@@ -208,6 +208,8 @@ trap terminate INT TERM EXIT
 
 if [[ "$ENVIRONMENT" == "singlenode-sqlserver" ]]; then
   EXTERNAL_SERVICES="hadoop-master sqlserver"
+elif [[ "$ENVIRONMENT" == "singlenode-ldap" ]]; then
+  EXTERNAL_SERVICES="hadoop-master ldapserver cassandra"
```

Comment:
With a change you suggested the build is failing. It looks like there are some tests which are using cassandra. 
Please see: https://s3.amazonaws.com/archive.travis-ci.org/jobs/265138394/log.txt?X-Amz-Expires=30&X-Amz-Date=20170817T051706Z&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJRYRXRSVGNKPKO5A/20170817/us-east-1/s3/aws4_request&X-Amz-SignedHeaders=host&X-Amz-Signature=5c03f331159887722fcb0025a8a434b624faa084ded9ed8c03084ead89ef7836